modules/claims_api/lib/claims_api/common/exceptions/lighthouse/timeout.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module Common
+    module Exceptions
+      module Lighthouse
+        class Timeout < StandardError
+          def initialize(errors)
+            @errors = [errors].flatten
+            super
+          end
+
+          def errors
+            errors_array = []
+            errors_array << {
+              status: status_code.to_s, # LH standards want this be a string
+              title: 'Upstream timeout',
+              detail: 'An upstream service timed out.'
+            }
+            errors_array
+          end
+
+          def status_code
+            504
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
@stiehlrod  Would it be valuable to add a test in the `lighthouse_error_handler_spec` file for this error?